File: 151002/Krumkachev/DC/DC/Models/Issue.cs
namespace DC.Models
{
	public class Issue : BaseModel
	{
		public long CreatorId { get; set; }

		public Creator Creator { get; set; } = null!;

		public string Title { get; set; } = string.Empty;

		public string Content { get; set; } = string.Empty;

		public DateTime Created { get; set; }

		public DateTime Modified { get; set; }

		public ICollection<Label> Labels { get; set; } = [];
	}
}

File: 151004/Terebey/Jpa/src/main/java/com/distributed_computing/jpa/bean/Issue.java
package com.distributed_computing.jpa.bean;

import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDateTime;
import java.util.List;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Entity(name = "tbl_issue")
public class Issue {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    int id;

    @ManyToOne
    @JoinColumn(name = "creatorId", referencedColumnName = "id")
    Creator creator;

    @OneToMany(mappedBy = "issue")
    List<Message> comments;

    @ManyToMany(mappedBy = "issue")
    List<Message> labels;

    String title;
    String content;
    LocalDateTime created;
    LocalDateTime modified;
}
